app/renderer/js/pages/preference/general-section.ts

```diff
@@ -354,30 +359,30 @@ class GeneralSection extends BaseSection {
 		});
 	}
 
-	customCssDialog() {
+	customCssDialog(): void {
 		const showDialogOptions = {
 			title: 'Select file',
 			defaultId: 1,
 			properties: ['openFile'],
 			filters: [{ name: 'CSS file', extensions: ['css'] }]
 		};
 
-		dialog.showOpenDialog(showDialogOptions, selectedFile => {
+		dialog.showOpenDialog(showDialogOptions as OpenDialogOptions, selectedFile => {
```

Comment:
See my comment above for the similar thing in previous commit.

Reply:
defaultId is not a property in OpenDialogOptions. Should i remove it?

Reply:
Yes, I mentioned this above for similar thing.